integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java

```diff
@@ -758,14 +757,6 @@ void testMultiClustersRollingRestart() {
         "domain event {0} to be logged",
         DOMAIN_CHANGED);
 
-    // verify the DomainProcessing Starting/Completed event is generated
```

Comment:
Even if we delay testing for a Completed event to a later JIRA, I'm worried that if we just delete these checks then we will have trouble finding where to restore them. I guess we will have the source control. Still, it might be easier to put a TO-DO type marker.

Reply:
We don't want the domain_processing_starting event any more. We do want domain-completed.